fifo/sim/BUILD.bazel

```diff
@@ -66,3 +80,32 @@ verilog_sim_test(
     top = "br_fifo_flops_push_credit_tb",
     deps = [":br_fifo_flops_push_credit_tb"],
 )
+
+verilog_library(
+    name = "br_fifo_ctrl_1r1w_tb",
+    srcs = ["br_fifo_ctrl_1r1w_tb.sv"],
+    deps = [
+        ":br_fifo_test_harness",
+        "@bedrock-rtl//delay/rtl:br_delay_valid",
+        "@bedrock-rtl//fifo/rtl:br_fifo_ctrl_1r1w",
+        "@bedrock-rtl//misc/sim:br_test_driver",
+        "@bedrock-rtl//ram/rtl:br_ram_flops_1r1w",
+    ],
+)
+
+verilog_elab_test(
+    name = "br_fifo_ctrl_1r1w_tb_elab_test",
+    deps = [":br_fifo_ctrl_1r1w_tb"],
+)
+
+verilog_sim_test(
+    name = "br_fifo_ctrl_1r1w_vcs_test",
+    defines = [
+        "SV_ASSERT_ON",
+        "BR_ENABLE_IMPL_CHECKS",
+    ],
+    opts = ["-assert global_finish_maxfail=1+offending_values"],
+    tool = "vcs",
+    top = "br_fifo_ctrl_1r1w_tb",
+    deps = [":br_fifo_ctrl_1r1w_tb"],
+)
```

Comment:
We need to test the FIFO with multiple parameter combinations. Please stamp multiple targets with different `params`

Reply:
That's being handled internally in the testbench.